### DISCUSSION TITLE
Homing

### DISCUSSION THREAD

**Discussion by DirtyDerv** [winder/Universal-G-Code-Sender#1832] [Help]
Ok so after I pulled the CNC out of mothballs (2 years it's been sat) I decided to use a grbl controller with ugs.  I have set up the machine and can get it to home. 

The issue I have is that it reports the home position to be x-185 y-85 z0.  The limits are at the front left and top of the z-axis.  I'm wondering if I can get the homed position to display 0 (or 1 as I have a 1 mm back off) 

Before I used to use Mach 3 but don't have a parallel port anymore.

**Comment by breiler** [comment]
Hi @DirtyDerv, you might find an answer here:
https://github.com/gnea/grbl/wiki/Frequently-Asked-Questions#why-is-grbl-in-all-negative-coordinates-after-homing-or-it-so-annoying-and-not-what-im-used-to

**Comment by DirtyDerv** [comment]
Cheers dude ill have to change my thinking a little lol, thanks for the reply
